backend/tests/helpers/UserHelper.test.js

```diff
@@ -0,0 +1,67 @@
+const { expect } = require( 'chai' );
+const { getUserListFromSessions } = require( '../../src/helpers/helpers' );
+
+describe( 'getUsers', function() {
```

Comment:
Use arrow function

Reply:
You commented an old version of the code :D

Reply:
Already done.